test/e2e/webhook-http-proxy-test

```diff
@@ -129,20 +117,28 @@ set -x
 helm "${emtp_helm_args[@]}"
 set +x
 
-aemm_helm_args=(
+anth_helm_args=(
   upgrade
   --install
-  $CLUSTER_NAME-aemm
-  $AEMM_DL_URL
+  $CLUSTER_NAME-anth
+  $SCRIPTPATH/../../config/helm/aws-node-termination-handler/
+  --force
+  --namespace kube-system
   --wait
-  --namespace default
-  --set servicePort="$IMDS_PORT"
+  --set instanceMetadataURL="http://$AEMM_URL:$IMDS_PORT"
+  --set image.repository="$NODE_TERMINATION_HANDLER_DOCKER_REPO"
+  --set image.tag="$NODE_TERMINATION_HANDLER_DOCKER_TAG"
+  --set enableSpotInterruptionDraining="true"
+  --set enableScheduledEventDraining="true"
+  --set webhookURL="$WEBHOOK_URL"
```

Comment:
update to:

```
webhookURL="${WEBHOOK_URL:-"http://localhost:$IMDS_PORT"}"
```

to support `eks-cluster-test`

Reply:
I removed all this crazy webhook stuff and reworked it to be passed from the driver like what happens in the eks driver.  I also moved the webhook to its own network namespace since there's no need for it to use host networking.